kubectl-cloudflow/config/config.go

```diff
@@ -165,12 +166,22 @@ func validateConfig(config *Config, applicationSpec cfapp.CloudflowApplicationSp
 	}
 
 	hoconConf := config.parse()
-
+	cloudflowConfig := hoconConf.GetConfig(cloudflowPath)
+	if cloudflowConfig != nil && cloudflowConfig.Root().IsObject() {
+		for section := range cloudflowConfig.Root().GetObject().Items() {
```

Comment:
Q: Why do we have an indentation of 8 chars in Go? Looks excessive.

Reply:
Indentation is actually one tab, I think. Somehow Github shows the tab as this wide.